### PR TITLE
Remove deprecated `IncludeRequestPayload` option

### PR DESCRIPTION
**Pull request by MartinZid**
Remove deprecated `IncludeRequestPayload` option from example and documentation, use `MaxRequestBodySize` instead.

### CODE REVIEW

File: src/platforms/dotnet/guides/aspnetcore/index.mdx
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnetcore/index.mdx:244]:
```suggestion
This parameter controls whether integrations should capture HTTP request bodies. It can be set to one of the following values:

- `never`: Request bodies are never sent.
- `small`: Only small request bodies will be captured. The cutoff for small depends on the SDK (typically 4KB).
- `medium`: Medium and small requests will be captured (typically 10KB).
- `always`: The SDK will always capture the request body as long as Sentry can make sense of it.

If the request bodies should be captured, all requests will have the `EnableRewind` method invoked. This is done so that the request data can be read later, in case an error happens while processing the request.
```

**Review by imatwawana — APPROVED** [review]
Made some minor wording suggestions

File: src/platforms/dotnet/guides/aspnetcore/index.mdx
Comment (MartinZid) [reply src/platforms/dotnet/guides/aspnetcore/index.mdx:244]: I do like these suggestions, however I copied this part of documentation for [here](https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/configuration/options/#max-request-body-size) (so it is consistent).

So if we commit your improvements, I should also make changes to the part of documentation linked above.

Comment (imatwawana) [reply src/platforms/dotnet/guides/aspnetcore/index.mdx:244]: Assuming @bruno-garcia has no objections, you're right, it should be updated in both spots. Let's wait for review from @bruno-garcia first. Thanks for pointing that out!

**Review by bruno-garcia — APPROVED** [review]
Great catch! Thanks a lot for this PR

File: src/platforms/dotnet/guides/aspnetcore/index.mdx
Comment (bruno-garcia) [reply src/platforms/dotnet/guides/aspnetcore/index.mdx:244]: Makes sense to align them both. Thanks for raising this

**Review by imatwawana — APPROVED** [review]
Looks great! Thanks so much!

**Pull request merged**
